readme.txt

```diff
@@ -37,6 +37,10 @@ Altapay's Payment Gateway for WooCommerce provides merchants with access to a fu
 
 == Changelog ==
 
+= 3.2.3 =
+* Update supported version
+* Minor label adjustments
+
```

Comment:
Please be more specific, example:
* Support version X of the gateway
* Update synch button label to 'Synchronize payment methods'

Reply:
@emicha I've updated the release notes according to the suggested changes.